### PR TITLE
better code to employ decaying learning rate

### PR DESCRIPTION
**Pull request by primetang**
I think `tf.train.exponential_decay` is a more better and more standard method than `sess.run(tf.assign(model.lr, args.learning_rate * (args.decay_rate ** e)))` to employ decaying learning rate.

### CODE REVIEW

**Comment by normanheckscher** [comment]
Can implement learning algorithm to learn the best decay rate?
https://arxiv.org/abs/1606.04474
